eng/Versions.props

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <!-- Production Dependencies -->
   <PropertyGroup>
-    <PrivateSourceBuildReferencePackagesPackageVersion>1.0.0-beta.20261.1</PrivateSourceBuildReferencePackagesPackageVersion>
-    <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-5.0.100-preview.1.20229.1</PrivateSourceBuiltArtifactsPackageVersion>
+    <PrivateSourceBuildReferencePackagesPackageVersion>1.0.0-beta.20329.1</PrivateSourceBuildReferencePackagesPackageVersion>
+    <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-5.0.100-bootstrap.3</PrivateSourceBuiltArtifactsPackageVersion>
```

Comment:
Are the contents of this documented? (Specifically, what's in it and why--I think I heard it was somewhat manual?)

Reply:
They're not documented.  It's an almagamation of previously source-built archive from 3.1 and some packages built in the production build for 5.0.  Once we get bootstrapping going, we can get a full update from 5.0.